site/content/en/docs/start/_index.md

```diff
@@ -49,7 +49,7 @@ curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-latest
 sudo rpm -ivh minikube-latest.x86_64.rpm
 ```
 
-### ARM
+### ARM64
```

Comment:
Should make it consistent, i.e. "arm64". Probably throw in the other names as well, they are used by the .rpm

Reply:
How about going with "arm64 / aarch64" and "amd64 / x86_64"? Actual artifact is named "minikube-latest.aarch64.rpm" on storage - same applies to amd64 - RPM artifact is named "x86_64", whereas both refer to the same architecture (arm64 and aarch64 to ARM64, x86_64 and arm64 to 64-bit version of x86 with AMD instruction set)

Reply:
Sounds good